internal/bcdb/worldstate_query_processor.go

```diff
@@ -162,15 +161,27 @@ func (q *worldstateQueryProcessor) executeJSONQuery(dbName, querierUserID string
 		}
 	}
 
-	keys, err := q.jsonQueryExecutor.ExecuteQuery(dbName, query)
+	snapshots, err := q.db.GetSnapshot(
+		[]string{
+			worldstate.DatabasesDBName,
+			dbName,
+			stateindex.IndexDB(dbName),
+		},
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	jsonQueryExecutor := queryexecutor.NewWorldStateJSONQueryExecutor(snapshots, q.logger)
```

Comment:
I didn't find the place where snapshot is released.

Reply:
Good catch. Done.